### PR TITLE
Add an introductory article on ShardingSphere-Proxy

### PR DESCRIPTION
**Pull request by longtai-cn**
Changes proposed in this pull request:
- Add an introductory article on ShardingSphere-Proxy.

### CODE REVIEW

File: docs/blog/content/material/proxyIntroduce.cn.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
++++
+title = "ShardingSphere-Proxy：从实际场景出发，快速上手"
+weight = 26
```
Comment (TeslaCN) [inline docs/blog/content/material/proxyIntroduce.cn.md:3]: The weight is duplicated with an existed article. Could you find the maximum weight in current blogs?

Comment (longtai-cn) [reply docs/blog/content/material/proxyIntroduce.cn.md:3]: @TeslaCN In the Chinese blog, 《[Apache ShardingSphere 在京东白条场景的落地之旅](https://shardingsphere.apache.org/blog/cn/material/oct_12_9_shardingsphere_jd-baitiao_story_of_an_implementation_journey/)》 has a maximum weight of 25, which I added on this basis

-- Is it according to the maximum weight of Chinese blog or the overall weight of blog.

Comment (TeslaCN) [reply docs/blog/content/material/proxyIntroduce.cn.md:3]: The weight shall be based on maximum of all blogs. Some articles may be translated into Chinese or English later.

Comment (longtai-cn) [reply docs/blog/content/material/proxyIntroduce.cn.md:3]: OK, I see

**Review by TeslaCN — APPROVED** [review]

**Comment by TeslaCN** [comment]
@longtai-cn Thank you!

**Pull request merged**